### DISCUSSION TITLE
Amount of Parameters in DALLE-PyTorch?

### DISCUSSION THREAD

**Discussion by sewageseaweed** [lucidrains/DALLE-pytorch#283] [General]
Hi all,

I found this github after reading OpenAI's blog post on DALL-E. In the post, they mentioned that their DALL-E model had 12 billion parameters. Do we have an idea of how many parameters PyTorch's DALL-E is?

Thank you!

**Comment by adamonkey** [comment]
It depends on how you configure your model. With the defaults, you get 20M parameters for the DALLE part. 

![image](https://user-images.githubusercontent.com/20653001/120835799-cd664380-c532-11eb-90a9-a51998e896a1.png)
